### DISCUSSION TITLE
Feature request: Localized metadata

### DISCUSSION THREAD

**Discussion by mtotschnig** [FortAwesome/Font-Awesome#20932] [Feature Requests]
### Is your feature request related to a problem?

I plan to include the free icons into an open source app and allow users to pick icons for categories. Usability would be immensely enhanced, if user were able to search icons by title. Using the icons.yml file, I can provide this functionality for English speaking users, but this would of course be of limited value for the rest of the world.
So my question is, if you would be willing to add a localization layer to the metadata pulling in translations from a project set up on Weblate. I would be able to contribute German and French, and might also be able to motivate some of the translators of my app for other languages to contribute to such a project.

### Feature description

The labels in icons.yml would be extracted into a file that can be used as translation source on Weblate. As soon as translations come available on Weblate, they are pulled into the same folder.

**Comment by AugustinMauroy** [comment]
+1
